client/pages/colors.jsx

```diff
@@ -0,0 +1,161 @@
+    if (this.state.colors[this.state.currentIndex].color === 'orange') return 'rgba(255, 152, 17, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'yellow') return 'rgba(255, 212, 34, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'green') return 'rgba(109, 200, 42, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'blue') return '';
+    if (this.state.colors[this.state.currentIndex].color === 'purple') return 'rgba(171, 109, 208, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'brown') return 'rgba(191, 125, 56, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'pink') return 'rgba(255, 110, 148, 0.8)';
+    if (this.state.colors[this.state.currentIndex].color === 'black') return 'rgba(0, 0, 0, 0.7)';
+    if (this.state.colors[this.state.currentIndex].color === 'white') return 'rgba(255, 255, 255, 0.8)';
+  }
+
+  textColor() {
+    if (this.state.colors[this.state.currentIndex].color === 'black') return 'white';
+    return '';
+  }
+
+  render() {
+    if (this.state.colors.length === 0) return null;
+    const { imageUrl } = this.state.colors[this.state.currentIndex];
+    const color = this.state.colors[this.state.currentIndex].color;
+    const { imageText } = this.state.colors[this.state.currentIndex];
+    const colorClass = this.bgColor();
+    const textColor = this.textColor();
+
+    const displayType = !this.state.wordShowing
+      ? <div className='style' style={{ backgroundColor: `${colorClass}`, color: `${textColor}` }}>
+        <div className="row">
+          <div className="column-third">
+            <i onClick={this.handleClick} className="fas fa-chevron-left"></i>
+          </div>
+          <div className="center-img row">
+            <span
+              id='image'
+              onClick={this.handleClick}>
+                {color}
+            </span>
+          </div>
+          <div className="column-third">
+            <i onClick={this.handleClick} className="fas fa-chevron-right"></i>
+          </div>
+        </div>
+        <div className='col-full text-align'>
+          <span className='word-text'></span>
+        </div>
+      </div>
+      : <div className='style' style={{ backgroundColor: `${colorClass}`, color: `${textColor}` }}>
+        <div className="row">
+          <div className="column-third">
+            <i onClick={this.handleClick} className="fas fa-chevron-left"></i>
+          </div>
+          <div className="center-img">
+            <img
+              id='color'
+              src={imageUrl}
+              onClick={this.handleClick}>
+            </img>
+          </div>
+          <div className="column-third">
+            <i onClick={this.handleClick} className="fas fa-chevron-right"></i>
+          </div>
+        </div>
+        <div className='col-full text-align'>
+          <span className='word-text'>{imageText}</span>
+        </div>
+      </div>;
```

Comment:
It looks like most of this is the same. Try to keep all of the same React elements, but only use a ternary expression in the spot(s) that need to be different based on whether the word is showing or not.